app/assets/stylesheets/components/_background.scss

```diff
@@ -0,0 +1 @@
+.bg-gray-lighter { background-color: $gray-lighter; }
```

Comment:
I'm guessing we may continue to add additional variations so I think having them all in `background.scss` makes sense.